packages/svelte/src/internal/client/render.js

```diff
@@ -2225,7 +2225,6 @@ function each(anchor_node, collection, flags, key_fn, render_fn, fallback_fn, re
 					block.d = null;
 				}
 				let anchor = block.a;
-				const is_controlled = (block.f & EACH_IS_CONTROLLED) !== 0;
```

Comment:
Can these flags change in between? 

Function already closes over `block` so this shouldn't really affect perf I think, lmk if it changes anything.

Reply:
This flag can change FWIW. It’s not static, and can change from transitions being enabled, that turn this off